docs/internal-developers/testing/releases/1070.md

```diff
@@ -0,0 +1,128 @@
+--- | ---
+![imatge](https://github.com/woocommerce/woocommerce-blocks/assets/3616980/62efe190-c3e6-4aa5-8cf6-a6882e0188f9) | ![imatge](https://github.com/woocommerce/woocommerce-blocks/assets/3616980/ca5fb734-9b1d-4f35-bd8b-cc0283da4761)
+
+### Mini Cart: Fix styling of the Start Shopping button in the drawer [#10169](https://github.com/woocommerce/woocommerce-blocks/pull/10169)
+
+1. Activate Storefront.
+2. Add the Mini-Cart block to a page.
+3. Visit the page on the frontend and select the (empty) Mini-Cart to toggle the drawer.
+4. Verify the button is displaying correctly.
+
+| Before | After |
+| ------ | ----- |
+|<img width="883" alt="before-2" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/9d8ca8a5-1662-4ebd-8233-3a2948e70843">|<img width="898" alt="after-2" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/f7088abc-b17b-42fa-b329-57cd401fbe79">|
+
+### Mini Cart block > Update block registration to rely on a metadata file. [#10168](https://github.com/woocommerce/woocommerce-blocks/pull/10168)
+
+1. Create a new post
+2. Make sure the mini-cart block is available for insertion
+3. Insert the block, make changes to its settings and save
+4. On the frontend, make sure everything works as expected: you can add/remove products from the cart and the style changes made in the editor are visible on the frontend.
+
+### Enhance: Bring back user to 1st page when filters are changed [#9952](https://github.com/woocommerce/woocommerce-blocks/pull/9952)
+
+1. Create a new post.
+2. Add all filter blocks and a 'Products Beta' block to it.
+3. Save the post and navigate to the frontend.
+4. Navigate to the 2nd page using pagination. 
+5. Modify any filter and confirm that the page number resets to the 1st page upon filter alteration. 
+6. Repeat the above steps and validate that this also holds true for:
+    - 'Product Collection' block (The product collection block is currently in the experimental phase and is not shipped with the plugin/core)
+    - 'Classic WooCommerce Product Grid Block'
+    - Note: This behavior doesn't affect the 'All Products' block.
+
+### Adjust Banner padding to improve mobile experience [#10107](https://github.com/woocommerce/woocommerce-blocks/pull/10107)
+
+1. Create a test page.
+2. Add the Banner pattern to it.
+3. Switch the preview to Mobile.
+4. See that the text looks like the after screenshot above.
+
+| Before | After |
+| ------ | ----- |
+| <img width="407" alt="Screenshot 2023-07-06 at 10 02 40" src="https://github.com/woocommerce/woocommerce-blocks/assets/186112/2eb84729-ecba-490c-99e2-9ce575b0b595"> | <img width="408" alt="Screenshot 2023-07-06 at 10 02 18" src="https://github.com/woocommerce/woocommerce-blocks/assets/186112/c935b846-3451-495a-94ee-27e2f2c7ae4a"> |
+
+### Single Product Template > Ensure extensions can't trigger fatal errors on templates without any post content blocks [#10128](https://github.com/woocommerce/woocommerce-blocks/pull/10128)
```

Comment:
Due to the requirement of adding code snippets, I think we should either remove this from the testing notes, or rewrite the instructions to do a regression test if possible, don't know if it is in this case though.

Reply:
Great, thank you for pointing this out, I just rewrote the testing instructions to remove the part requiring to modify the plugin code